### ISSUE TITLE
tour: "underlying array" instead of "slice"?

### ISSUE THREAD

**Issue by jensdpersson** [golang/tour#821]
Context: https://tour.golang.org/moretypes/10

"The default is zero for the low bound and the length of the slice for the high bound."

Should not this be 

"The default is zero for the low bound and the length of the underlying array for the high bound."

**Comment by crisman** [comment]
No, slices can have a length that is smaller than the slice's capacity.  The next slide has some useful context:
https://go.dev/tour/moretypes/11
See also the slice forming examples in:
https://go.dev/tour/moretypes/7

**Comment by crisman** [comment]
Ah, I see the confusion now.

Duplicate of golang/tour#182
